### ISSUE TITLE
Catastrophic failure: System.ArgumentException: Argument was empty (Parameter 'settings.SerializedTestCases')

### ISSUE THREAD

**Issue by gieniowski** [xunit/visualstudio.xunit#417]
When running `dotnet test --filter aaaa` (filter should not have match in any test name) I have following errror:

```
Catastrophic failure: System.ArgumentException: Argument was empty (Parameter 'settings.SerializedTestCases')
   at Xunit.Internal.Guard.ArgumentNotNullOrEmpty[T](T argValue, String argName)
   at Xunit.Runner.v3.Xunit3.Run(IMessageSink messageSink, FrontControllerRunSettings settings)
   at Xunit.XunitFrontController.Run(IMessageSink messageSink, FrontControllerRunSettings settings)
   at Xunit.Runner.VisualStudio.VsTestRunner.RunTestsInAssembly(IRunContext runContext, IFrameworkHandle frameworkHandle, LoggerHelper logger, TestPlatformContext testPlatformContext, RunSettings runSettings, IMessageSink reporterMessageHandler, AssemblyRunInfo runInfo)
   at Xunit.Runner.VisualStudio.VsTestRunner.RunTestsInAssembly(IRunContext runContext, IFrameworkHandle frameworkHandle, LoggerHelper logger, TestPlatformContext testPlatformContext, RunSettings runSettings, IMessageSink reporterMessageHandler, AssemblyRunInfo runInfo)
   at Xunit.Runner.VisualStudio.VsTestRunner.RunTestsInAssembly(IRunContext runContext, IFrameworkHandle frameworkHandle, LoggerHelper logger, TestPlatformContext testPlatformContext, RunSettings runSettings, IMessageSink reporterMessageHandler, AssemblyRunInfo runInfo)
```

I am using these packages in the test project:
```
    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.11.1" />
    <PackageReference Include="xunit.runner.visualstudio" Version="3.0.0-pre.30">
      <PrivateAssets>all</PrivateAssets>
      <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
    </PackageReference>
    <PackageReference Include="xunit.v3" Version="0.3.0-pre.18" />
```

Here's my os and dotnet version:
```
dotnet --info
.NET SDK:
 Version:           9.0.100-rc.1.24452.12
 Commit:            81a714c6d3
 Workload version:  9.0.100-manifests.67cd1eb6
 MSBuild version:   17.12.0-preview-24422-09+d17ec720d

Runtime Environment:
 OS Name:     Windows
 OS Version:  10.0.22631
 OS Platform: Windows
 RID:         win-x64
 Base Path:   C:\Program Files\dotnet\sdk\9.0.100-rc.1.24452.12\
 ```

**Comment by bradwilson** [comment]
Fix available in `3.0.0-pre.34` https://xunit.net/docs/using-ci-builds

**Comment by bradwilson** [comment]
Post fix:

```
$ dotnet test test\test.v3 --framework net472 --filter aaaa
  Determining projects to restore...
  All projects are up-to-date for restore.
  xunit.runner.visualstudio -> C:\Dev\xunit\visualstudio\src\xunit.runner.visualstudio\bin\Debug\net472\xunit.runner.visualstudio.testadapter.dll
  xunit.runner.visualstudio.testadapter -> C:\Dev\xunit\visualstudio\src\xunit.runner.visualstudio\bin\Debug\net472\merged\xunit.runner.visualstudio.testadapter.dll
  test.v3 -> C:\Dev\xunit\visualstudio\test\test.v3\bin\Debug\net472\test.v3.exe
Test run for C:\Dev\xunit\visualstudio\test\test.v3\bin\Debug\net472\test.v3.exe (.NETFramework,Version=v4.7.2)
VSTest version 17.11.0 (x64)

Starting test execution, please wait...
A total of 1 test files matched the specified pattern.
No test matches the given testcase filter `aaaa` in C:\Dev\xunit\visualstudio\test\test.v3\bin\Debug\net472\test.v3.exe
```
